### DISCUSSION TITLE
How do I interract with the component buttons while coding?

### DISCUSSION THREAD

**Discussion by deok420** [ppy/osu#20996] [Q&A] [answered]
I cannot find the skin editor component buttons anywhere in the code. I've looked everywhere they might be, but to no avail. I even looked on the github to see if anyone has ever mentioned them, but no. I'm trying to interact with every individual component button, but I have no idea what the name of the button is in the code. Sorry if this is a very rookie question, but I'm not used to working on other people's projects. 

If you don't know which buttons I'm talking about, it's these ones: https://prnt.sc/7wmHsqxCaz3z

**Comment by mk56-spn** [comment, marked as answer]
![toolboxbutton](https://user-images.githubusercontent.com/74463310/198640005-0febc144-8169-4648-acdc-d81a8f0fb39c.png)

Its this button, I recommend using the Draw Visualizer  (ctrl + f1) to find this sort of thing 

![osu_2022-10-28_16-18-38](https://user-images.githubusercontent.com/74463310/198640682-8d6eec94-eb13-45b1-9177-b84f42a4aab3.png)

**Reply by deok420** [reply]
You're a life saver, thank you so much!
